### ISSUE TITLE
YOLOv11-medium default training setting

### ISSUE THREAD

**Issue by prodigiouscognition** [ultralytics/ultralytics#21247]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi,
When we train yolo v11 medium model (the detector), the mAP mentioned on the ultralytics website happens to be 51.5 [](([](https://docs.ultralytics.com/tasks/detect/))) on coco.
However, when I train the model using the default model setting on the same dataset, the mAP values don't match, and the mAP remains slightly low.
I wanted to know what setting was used when it was trained originally.
To reproduce the results, what hyperparameters should be used?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @prodigiouscognition, thank you for your interest in Ultralytics 🚀! This is an automated response to help you get started. An Ultralytics engineer will also assist you here soon.

We recommend checking out our [Docs](https://docs.ultralytics.com/) for comprehensive guidance. You can find detailed information on [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage, as well as commonly asked questions.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us investigate your issue more efficiently.

If you have a ❓ Question about custom training, please share as many details as possible, such as dataset image samples and training logs. Also, make sure to review our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Feel free to join our community where it suits you best! For real-time chat, hop onto [Discord](https://discord.com/invite/ultralytics) 🎧. For longer discussions, visit [Discourse](https://community.ultralytics.com/). You can also join conversations on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to connect with other users.

## Upgrade

Please ensure you are using the latest `ultralytics` package and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in any of the following up-to-date, verified environments (with all dependencies such as [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thanks for reaching out and for helping improve Ultralytics!

**Comment by Y-T-G** [comment]
You can check this

https://github.com/ultralytics/ultralytics/issues/17048#issuecomment-2424742030
